crawlera_fetch/middleware.py

```diff
@@ -178,19 +181,23 @@ def process_response(self, request: Request, response: Response, spider: Spider)
                 logger.warning(log_msg)
                 return response
 
-        if json_response.get("crawlera_error"):
-            error = json_response["crawlera_error"]
-            message = json_response["body"]
+        server_error = json_response.get("crawlera_error") or json_response.get("error_code")
+        original_status = json_response.get("original_status", response.status)
+        request_id = json_response.get("id") or json_response.get("uncork_id")
+        if server_error:
+            message = json_response.get("body") or json_response.get("message")
             self.stats.inc_value("crawlera_fetch/response_error")
-            self.stats.inc_value("crawlera_fetch/response_error/{}".format(error))
+            self.stats.inc_value("crawlera_fetch/response_error/{}".format(server_error))
             log_msg = (
-                "Error downloading <{} {}> (Original status: {}, Fetch API error message: {})"
+                "Error downloading <{} {}> (Original status: {}, Fetch API error message: {}). "
+                "Request ID: {}"
```

Comment:
Nitpick: what do you think about putting the request id inside the parenthesis? Or maybe removing them altoguether: `Error downloading <GET https://example.org> - Original status: 503, Fetch API error message: Server busy: too many outstanding requests, Request ID: None`